### PR TITLE
Rename to camelCase

### PR DESCRIPTION
**Pull request by joshuataylor**
Ready for review

### CODE REVIEW

File: modules/line_item/src/Entity/CommerceLineItem.php [hunk 1]
```diff
@@ -146,7 +146,7 @@ public function getOwner() {
    * {@inheritdoc}
    */
   public function getOwnerId() {
-    return $this->get('uid')->target_id;
+    return $this->get('uid')->targetId;
```
Comment (bojanz) [inline modules/line_item/src/Entity/CommerceLineItem.php]: We don't change field names, they're in the database (and this one is provided by core).

File: modules/line_item/src/Entity/CommerceLineItem.php [hunk 2]
```diff
@@ -82,12 +82,12 @@ public function preSave(EntityStorageInterface $storage) {
...
     parent::preSaveRevision($storage, $record);
 
-    if (!$this->isNewRevision() && isset($this->original) && (!isset($record->revision_log) || $record->revision_log === '')) {
+    if (!$this->isNewRevision() && isset($this->original) && (!isset($record->revisionLog) || $record->revisionLog === '')) {
       // If we are updating an existing line item without adding a new revision, we
-      // need to make sure $entity->revision_log is reset whenever it is empty.
+      // need to make sure $entity->revisionLog is reset whenever it is empty.
       // Therefore, this code allows us to avoid clobbering an existing log
       // entry with an empty one.
-      $record->revision_log = $this->original->revision_log->value;
```
Comment (bojanz) [inline modules/line_item/src/Entity/CommerceLineItem.php:90]: Can't change revision_log, it's a field.

File: modules/order/src/Entity/CommerceOrder.php [hunk 3]
```diff
@@ -93,12 +93,12 @@ public function preSave(EntityStorageInterface $storage) {
...
 
-    if (!$this->isNewRevision() && isset($this->original) && (!isset($record->revision_log) || $record->revision_log === '')) {
+    if (!$this->isNewRevision() && isset($this->original) && (!isset($record->revisionLog) || $record->revisionLog === '')) {
       // If we are updating an existing order without adding a new revision, we
-      // need to make sure $entity->revision_log is reset whenever it is empty.
+      // need to make sure $entity->revisionLog is reset whenever it is empty.
       // Therefore, this code allows us to avoid clobbering an existing log
       // entry with an empty one.
-      $record->revision_log = $this->original->revision_log->value;
+      $record->revisionLog = $this->original->revisionLog->value;
```
Comment (bojanz) [inline modules/order/src/Entity/CommerceOrder.php]: Nope.

**Comment by joshuataylor** [comment]
Ah fair, changing now.

File: modules/product/src/Entity/CommerceProduct.php [hunk 4]
```diff
@@ -158,7 +158,7 @@ public function setOwner(UserInterface $account) {
    * {@inheritdoc}
    */
   public function getOwnerId() {
-    $this->get('uid')->target_id;
+    $this->get('uid')->targetId;
```
Comment (bojanz) [inline modules/product/src/Entity/CommerceProduct.php]: Nope.

File: modules/product/src/Entity/CommerceProduct.php [hunk 5]
```diff
@@ -175,18 +175,18 @@ public function setOwnerId($uid) {
   public function preSaveRevision(EntityStorageInterface $storage, \stdClass $record) {
     parent::preSaveRevision($storage, $record);
 
-    if (!$this->isNewRevision() && isset($this->original) && (!isset($record->revision_log) || $record->revision_log === '')) {
+    if (!$this->isNewRevision() && isset($this->original) && (!isset($record->revisionLog) || $record->revisionLog === '')) {
       // If we are updating an existing product without adding a new
       // revision and the user did not supply a revision log, keep the existing
       // one.
-      $record->revision_log = $this->original->getRevisionLog();
+      $record->revisionLog = $this->original->getRevisionLog();
```
Comment (bojanz) [inline modules/product/src/Entity/CommerceProduct.php]: Nope.

File: modules/price/src/Plugin/Field/FieldWidget/PriceSimpleWidget.php [hunk 6]
```diff
@@ -48,9 +48,9 @@ public function formElement(FieldItemListInterface $items, $delta, array $elemen
     $element['currency_code'] = array(
       '#type' => 'select',
       '#title' => $this->t('Currency code'),
-      '#default_value' => isset($items[$delta]->currency_code) ? $items[$delta]->currency_code : NULL,
+      '#default_value' => isset($items[$delta]->currencyCode) ? $items[$delta]->currencyCode : NULL,
```
Comment (bojanz) [inline modules/price/src/Plugin/Field/FieldWidget/PriceSimpleWidget.php]: currency_code is the column of the field here, can't rename it.

**Comment by joshuataylor** [comment]
Routing?
`path: '/admin/commerce/config/store/add/{commerceStoreType}'`

OR

`path: '/admin/commerce/config/store/add/{commerce_store_type}'`

**Comment by bojanz** [comment]
It needs to be commerce_store_type because that's the entity type id.

File: modules/line_item/src/Controller/CommerceLineItemController.php [hunk 7]
```diff
@@ -65,14 +65,14 @@ public function add(CommerceLineItemTypeInterface $commerce_line_item_type) {
...
-   * @param \Drupal\commerce_line_item\CommerceLineItemTypeInterface $commerce_line_item_type
+   * @param \Drupal\commerce_line_item\CommerceLineItemTypeInterface $commerceLineItemType
    *   The current line item type.
    *
    * @return string
    *   The page title.
    */
-  public function addPageTitle(CommerceLineItemTypeInterface $commerce_line_item_type) {
-    return $this->t('Create @label', array('@label' => $commerce_line_item_type->label()));
+  public function addPageTitle(CommerceLineItemTypeInterface $commerceLineItemType) {
```
Comment (bojanz) [inline modules/line_item/src/Controller/CommerceLineItemController.php]: Have you confirmed manually that this works? I seem to recall the $commerce_line_item_type variable being magical, that's why I couldn't kill the commerce_ prefix.

File: modules/price/config/schema/commerce_price.schema.yml [hunk 8]
```diff
@@ -2,7 +2,7 @@ commerce_price.commerce_currency.*:
   type: config_entity
   label: 'Currency'
   mapping:
-    currencyCode:
+    currency_code:
```
Comment (bojanz) [inline modules/price/config/schema/commerce_price.schema.yml]: This looks accidental.

File: modules/product/src/CommerceProductInterface.php [hunk 9]
```diff
@@ -126,6 +126,6 @@ public function getRevisionLog();
    * @return \Drupal\commerce_product\CommerceProductInterface
    *   The class instance that this method is called on.
    */
-  public function setRevisionLog($revision_log);
+  public function setrevision_log($revision_log);
```
Comment (bojanz) [inline modules/product/src/CommerceProductInterface.php]: Whoops

File: modules/product/src/Entity/CommerceProduct.php [hunk 10]
```diff
@@ -349,7 +349,7 @@ public function getRevisionLog() {
   /**
    * {@inheritdoc}
    */
-  public function setRevisionLog($revision_log) {
+  public function setrevision_log($revision_log) {
```
Comment (bojanz) [inline modules/product/src/Entity/CommerceProduct.php]: Also whoops

**Comment by bojanz** [comment]
We shouldn't do the currencyCode -> currency_code conversion, CMI properties will need some additional love, after I figure out the exact plan.

**Comment by bojanz** [comment]
Big thank you to you Josh, this is a lot of work.

**Pull request merged**